todo/tests/test_views.py

```diff
@@ -133,6 +133,64 @@ def test_no_javascript_in_task_note(todo_setup, client):
     assert task.note == bleach.clean(note, strip=True)
 
 
+@pytest.mark.django_db
+def test_completed_unchanged(todo_setup, client):
```

Comment:
Thanks for all the work you put into the test. So that your work is not wasted, we'll keep the test for sure.